src/main.ts

```diff
@@ -17,10 +17,13 @@ export async function run(): Promise<void> {
         console.log("No vulnerabilities were detected in the container image");
     } else {
         const errors = utils.extractErrorsFromLogs(trivyHelper.getTrivyLogPath(), trivyHelper.trivyToolName);
+        
+        console.log(errors);
```

Comment:
This is redundant. As `core.error` prints out the errors to the error stream.

Reply:
Removed.